src/components/VerificationForm/VerificationForm.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState, ChangeEvent, FC, CSSProperties  } from "react";
+import { Button, FormControl, Box } from "@mui/material";
+
+import PermIdentityIcon from "@mui/icons-material/PermIdentity";
```

Comment:
you could join the icons in one line, remind me to tell you how later 😋

Reply:
yes my lady